eclipse-collections/src/main/java/org/eclipse/collections/impl/multimap/bag/AbstractMutableBagMultimap.java

```diff
@@ -129,4 +129,10 @@ public void putOccurrences(K key, V value, int occurrences)
             this.addToTotalSize(occurrences);
         }
     }
+
+    @Override
+    public MutableBagMultimap<K, V> asSynchronized()
+    {
+        return SynchronizedBagMultimap.of(this);
```

Comment:
This is O(n) operation, the convention is that if a method is as<Something>() it should be O(1).